src/TypeScript/Analyzer/TypeScriptAnalyzer.cs

```diff
@@ -150,6 +150,10 @@ internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, Ca
                     consumer.Accept(filePath, issues);
                 }
             }
+            catch (EslintBridgeClientNotInitializedException)
```

Comment:
The analysis will fail but the next call will succeed.
I'll handle it better as part of #2190